include/nbl/video/utilities/IUtilities.h

```diff
@@ -366,6 +364,7 @@ class NBL_API IUtilities : public core::IReferenceCounted
             assert(cmdbuf->isResettable());
             assert(cmdpool->getQueueFamilyIndex() == queue->getFamilyIndex());
             assert(cmdbuf->getRecordingFlags().hasFlags(IGPUCommandBuffer::EU_ONE_TIME_SUBMIT_BIT));
+            assert(bufferRange.buffer->getCachedCreationParams().usage.hasFlags(asset::IBuffer::EUF_TRANSFER_DST_BIT));
```

Comment:
again don't assert, just return nullptr, we'll have matching errors in Release and Debug